design/sketch.webidl

```diff
@@ -266,6 +266,27 @@ interface WebGPUBindGroup {
 // PIPELINE CREATION (blend state, DS state, ..., pipelines)
 // ****************************************************************************
 
+// RasterizationState
+enum WebGPUFrontFace {
+    "ccw",
```

Comment:
I think these should be captialized because they are acronyms. CW and CCW

Reply:
Yep. Lowercase is typical in other Web APIs.